__tests__/kata6.joinNames.test.js

```diff
@@ -2,6 +2,7 @@ const { joinNames } = require('../src');
 
 describe('joinNames', () => {
   it('returns string of names, seperated by commas and an ampersand', () => {
+    // review
```

Comment:
It isn't recommended that you structure objects like this. Either go for all on one line (if it won't result in the line length exceeding 80 characters) or bring the key names onto their own lines:

```js
expect(joinNames([{
  name: 'Bart'
}, {
  name: 'Lisa'
}, {
  name: 'Maggie'
}]))
```